gulpfile.js

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
`bitcore-build` was created for bitcore submodules. It focuses on browserify the code, run tests on node and browser, and follow the submodules release strategy. I could include it and replace the definition of `test` and `release` tasks.